payjoin/src/send/mod.rs

```diff
@@ -211,7 +217,7 @@ impl<'a> RequestBuilder<'a> {
     pub fn build_recommended(
         self,
         min_fee_rate: FeeRate,
-    ) -> Result<RequestContext<'a>, CreateRequestError> {
```

Comment:
whats the motivation for removing this? is it the Mutex that holding the store?

Reply:
In order to persist data it needs to all be owned by the same struct so that it can be deserialized. Otherwise, the serialized format would need to reference another part of data, which is what this `'a` lifetime signified, that `uri: PjUri<'a>` was in memory in another part of the program.